src/main/java/bit/javaoop/Manager.java

```diff
@@ -0,0 +1,31 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+
+public abstract class Manager extends AbstractEmployee {
+
+    public ArrayList<AbstractEmployee> listOfEmployees;
+
+    public Manager(String name, double salary, ArrayList<AbstractEmployee> listOfEmployees) {
+        super(name, salary);
+        this.listOfEmployees = new ArrayList<>();
+
+    }
+
+    public abstract boolean canHire();
+
+    public abstract void hire(AbstractEmployee employee);
+
+    @Override
+    public String toString(int line) {
+        String result = new String();
+        for (int i = 0; i < line; i += 1)
+            result += '\t';
+        result += getName() + " - Manager\n";
+        for (AbstractEmployee e : listOfEmployees) {
+            result += e.toString(line + 1);
+        }
+        return result;
```

Comment:
- Raz to StringBuildery
- Dwa logika związana ze wstawianiem tabów powinna znaleźć się w klasie Company, każda klasa odpowiada za siebie i powinna realizować swoje odpowiedzialności tutaj w toStringu ładnie się wypisać.
Ładnie się wypisać z punktu siedzenia Managera to znaczy wypisać np. swoje imie + stanowisko( manager ) ew. salary
Natomiast z punktu siedzenia Company jest ładnie wypisać swoją strukturę tak aby było widać kto jest pod kim i na jakim stanowisku tutaj za to odpowiadały taby.